<?xml version="1.0"?>
    <!--
    
   -->
<launch>

    <arg name="robot_id" default="4"/>
    <arg name="loc_method" default="kollmorgen" />

    <!-- derived args -->
    <arg name="prefix" value="robot$(arg robot_id)"/>
    
    <!-- the group -->
    <group ns="$(arg prefix)" if="$(eval arg('loc_method') == 'NDT')">

      <!--  Node-->
      <node name="init_pose_manager" pkg="iliad_init_pose_manager" type="init_pose_manager.py" output="screen">
        <param name="robot_id" value="$(arg robot_id)"/>
        <param name="initial_pose_topic_name" value="/$(arg prefix)/initialpose"/>
        <param name="robot_pose_topic_name" value="/$(arg prefix)/mcl_pose_estimate"/>
        <param name="saved_pose_uri" value="$(env HOME)/.ros/lastKnownPose.json"/>
      </node>

    </group>

</launch>
